pkg/api/v1alpha1/cluster.go

```diff
@@ -465,16 +465,28 @@ func validateWorkerNodeGroups(clusterConfig *Cluster) error {
 	return nil
 }
 
-func validateAutoscalingConfig(autoscalingConfig *AutoScalingConfiguration) error {
-	if autoscalingConfig == nil {
+func validateAutoscalingConfig(w *WorkerNodeGroupConfiguration) error {
+	if w == nil {
 		return nil
 	}
-	if autoscalingConfig.MinCount < 0 {
+	if w.AutoScalingConfiguration == nil && *w.Count < 0 {
```

Comment:
isn't this already checked in line 429?

Reply:
Ah, I meant to pull it out and move it down here.

Do you think this line better belongs in `validateAutoscalingConfiguration` or `validateWorkerNodeGroups`?

Reply:
i dont have a strong preference :0